tests/exec/execute.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func ExecuteCommandOnPod(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, containerName string, command []string) (string, error) {
-	stdout, stderr, err := ExecuteCommandOnPodV2(virtCli, pod, containerName, command)
+	stdout, stderr, err := ExecuteCommandOnPodWithResults(virtCli, pod, containerName, command)
```

Comment:
I think we should remove `ExecuteCommandOnPod` as it seems redundant.

Reply:
Yes, I agree, it was done in #7944 but I had to close this PR since it had issues after doing it (I commented about it here https://github.com/kubevirt/kubevirt/pull/7944#issuecomment-1233074966).

Since it has too many changes in PR (moving things around and refactoring them) and after a discussion with @brybacki about it, I opened this PR just to cleanup utils (just to move execute functions to a new package).